openwisp_controller/config/static/config/css/admin.css

```diff
@@ -249,6 +249,34 @@ div.tabs-divider{
 .submit-row .previewlink:hover,
 .submit-row .previewlink:focus { background: #576a7c }
 
+.system-context{
+  overflow: hidden;
+}
+
+.hide-sc{
+  height: 150px;
+}
+
+.system-context-btn{
+  padding: 5px 25px;
+  margin-top: 5px;
+  text-transform: capitalize;
+  font-size: 13px;
+  border: 1px solid rgba(0, 0, 0, 0.1) !important;
+  background: #E6E6E6;
+  cursor: pointer;
+  display: none;
+}
```

Comment:
in general, you should not add a different style to a new button.
There are already buttons in the application, you should just reuse the style already used in the other buttons and make sure they look identical.

Just reuse the `.button` class and adjust padding/spacing if needed.